lib/graphql/plug/endpoint.ex

```diff
@@ -41,8 +39,22 @@ defmodule GraphQL.Plug.Endpoint do
     |> halt
   end
 
-  defp execute(conn, schema, query) do
-    case GraphQL.execute(schema, query) do
+  defp parse(query) do
```

Comment:
Shouldn't need this any more as `GraphQL` module only exposes `execute` function